ni_measurementlink_service/_internal/stubs/session_pb2.py

```diff
@@ -19,7 +19,6 @@
 _builder.BuildMessageAndEnumDescriptors(DESCRIPTOR, globals())
 _builder.BuildTopDescriptorsAndMessages(DESCRIPTOR, "session_pb2", globals())
 if _descriptor._USE_C_DESCRIPTORS == False:
-
     DESCRIPTOR._options = None
```

Comment:
FYI, I'm going to update generate_grpc_stubs.py to stop reformatting with Black. We already exclude the stubs in our Black and ni-python-styleguide configurations.

Reply:
https://github.com/ni/measurementlink-python/pull/247